hilti/runtime/src/tests/stream.cc

```diff
@@ -960,25 +960,25 @@ TEST_CASE("View") {
 
         SUBCASE("1") {
             Byte dst[1] = {'0'};
-            CHECK_EQ(v.extract(dst), "234567890"_b);
+            CHECK_EQ(v.extract(dst, 1), "234567890"_b);
```

Comment:
Let's get rid of the magic value here and below,

```suggestion
            CHECK_EQ(v.extract(dst, sizeof(dst) / sizeof(dst[0])), "234567890"_b);
```